charts/internal/calico/templates/node-cpva/deployment-calico-node-vertical-autoscaler.yaml

```diff
@@ -28,7 +28,11 @@ spec:
       {{- end }}
       priorityClassName: system-cluster-critical
       securityContext:
+        runAsNonRoot: true
         runAsUser: 65534
+        fsGroup: 1
+        supplementalGroups:
+        - 1
```

Comment:
Why did you change this as well?
Setting `runAsNonRoot` seems a bit redundant, but will not hurt. Hence, I have no problem with it.
However, I do not get why you add `fsGroup` and `supplementalGroups`.

Reply:
These were part of the PSP under `MustRunAs`, but there it was actually a specified range, so I looked at the actual components running, and found these values there. Aren't these required? Can we just drop these fields? How does it work when the PSP is not present anymore?

Reply:
My understanding of podsecuritypolicy was off. Your change is good.